### ISSUE TITLE
Plug-in support for Darwin armv8.

### ISSUE THREAD

**Issue by chrislovecnm** [jenkins-x/jx#8130]
I am noticing that a lot of plugins do not support the Mac running the m1 with the armv8 architecture. Is there work underway to get this fixed the across all plugins?

**Comment by tomhobson** [comment]
Hey @chrislovecnm, Yep work is ongoing to support this architecture. If you have a list of plugins that aren't yet working that would be a great thing for us to have!

**Comment by chrislovecnm** [comment]
Failed:

```
jx ui
jx pipeline log
jx gitops
jx admin
jx preview
jx secret
jx promote
jx application
jx semanticcheck
jx project
jx slack
```

I am thinking that none of the plugins in https://github.com/jenkins-x-plugins/ are supporting m1 on the mac

**Comment by chrislovecnm** [comment]
I am getting two different forms of errors

```
$ jx promote
ERROR: failed to install binary plugin jx-promote version 0.2.5 to /Users/clove/.jx3/plugins/bin: unable to locate binary for arm64 darwin for promote
```

```
$ jx semanticcheck
ERROR: failed to load plugin jx-semanticcheck: can't find latest version of plugin: {"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/repos#get-the-latest-release"}
```

**Comment by ankitm123** [comment]
The first error is because jx promote arm binary is missing.
The second error is because semantic check is a new plugin that has not been added to jx yet.

We are fixing our infrastructure atm, once that is done, we will roll out arm binaries, although adding support for kubernetes 1.22 is higher priority (we are close)

**Comment by chrislovecnm** [comment]
Do we have instructions on how to install a dev version of the plugin?  I have the binary building locally, but I cannot get jx to find it.

**Comment by chrislovecnm** [comment]
@ankitm123 I just put in this https://github.com/jenkins-x/jx-helpers/pull/369 - nothing is working without this PR

**Comment by luctrate** [comment]
@ankitm123 
thanks for communicating your current priority, i was a bit confused by the slow progress for m1 support. 
i have taken another look at the build, release files for jx-preview and opened another PR, where i updated the goreleaser version within the github action file. 
Please take a look as soon as your priorities allow it.
https://github.com/jenkins-x-plugins/jx-preview/pull/356
Its almost the last difference between the jx parent project, where darwin arm binaries are being build.

**Comment by ankitm123** [comment]
I think jx ui and jx slack are the only one remaining atm. jx semantic check needs to be added to jx first. @tomhobson can we add it now (since ur team is working on it ..)
